cmd/draconctl/pipelines/build.go

```diff
@@ -71,8 +76,10 @@ func buildPipeline(cmd *cobra.Command, args []string) error {
 	if err != nil {
 		return fmt.Errorf("could not initialise backend: %w", err)
 	}
-
-	pipeline, err := k8sBackend.Generate()
+	ops := pipelines.GenerationOpts{
+		ImagePullPolicy: (*corev1.PullPolicy)(&overrideImagePullPolicy),
```

Comment:
There is no check over here if the `overrideImagePullPolicy` is set to an accepted value. Also this is more complex than needed. You could do `&corev1.PullPolicy(overrideImagePullPolicy)`